lib/os/reboot.c

```diff
@@ -4,16 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * @file common target reboot functionality
- *
- * @details See misc/Kconfig and the reboot help for details.
- */
-
 #include <kernel.h>
-#include <drivers/timer/system_timer.h>
 #include <sys/printk.h>
-#include <sys/reboot.h>
```

Comment:
It may work without inclusion of this header, but shouldn't we still keep it because this file implements `sys/reboot.h`?

Reply:
We can keep it, but in this particular case it is not strictly necessary.